views/about.lavender

```diff
@@ -38,16 +38,16 @@ body
                 a(href='#') Guide Home
               li.dropdown-header Installation
               li
-                a(href="https://github.com/golavender/lavender", title="GitHub") via Composer
+                a(href="/guide.html#composer", title="GitHub") via Composer
```

Comment:
Thanks for weighing in on this one.

On Thursday, January 29, 2015, Thomas Woodward <notifications@github.com>
wrote:

> In views/about.lavender
> <https://github.com/golavender/golavender.github.io/pull/16#discussion_r23778013>
> :
>
> > @@ -38,16 +38,16 @@ body
> >                  a(href='#') Guide Home
> >                li.dropdown-header Installation
> >                li
> > -                a(href="https://github.com/golavender/lavender", title="GitHub") via Composer
> > +                a(href="/guide.html#composer", title="GitHub") via Composer
>
> you should either use or not use leading slashes throughout the site
> consistently. for the record on the internet it would be more correct to
> have the leading slashes, but with the way you guys are developing that
> just doesn't work locally
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/golavender/golavender.github.io/pull/16/files#r23778013>
> .
>


-- 
Dan